### PR TITLE
[BACKLOG-35131] Add Azure parser for AzureHDInsights specific scheme's

### PR DESCRIPTION
**Pull request by renjith-nem**
Updated to enable file system parser to list and select files on Azure HDInsights shim jobs

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 6s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl impl/vfs/hdfs
```
---
:ok_hand: All tests passed!
 
Tests run: **45**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-plugin/job/PR-2174/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 13s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl impl/vfs/hdfs
```
---
:ok_hand: All tests passed!
 
Tests run: **45**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-plugin/job/PR-2174/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl impl/vfs/hdfs
```
---
:ok_hand: All tests passed!
 
Tests run: **45**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-plugin/job/PR-2174/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by lucboudreau — APPROVED** [review]

File: impl/vfs/hdfs/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 1]
```diff
@@ -21,25 +22,25 @@
...
+    <argument ref="namedClusterService"/>
+    <argument ref="maprfsFileNameParser" />
+    <argument value="escalefs" type="java.lang.String"/>
     <argument ref="metastoreLocatorOsgi" />
   </bean>
 
   <bean class="org.pentaho.big.data.impl.vfs.hdfs.HDFSFileProvider" scope="singleton">
     <argument ref="hadoopFileSystemService"/>
     <argument ref="namedClusterService"/>
-    <argument ref="maprfsFileNameParser" />
```
Comment (rmansoor) [inline impl/vfs/hdfs/src/main/resources/OSGI-INF/blueprint/blueprint.xml:34]: why did you remove the maprfsFileNameParser here

Comment (renjith-nem) [reply impl/vfs/hdfs/src/main/resources/OSGI-INF/blueprint/blueprint.xml:34]: erm, it's not removed, it's been replaced with the Azure parser for WASB and ABFS scheme's which are Azure HDI Shim specific.
